Ginger/GingerCoreNET/Run/GingerRunner.cs

```diff
@@ -2361,7 +2372,15 @@ private bool GotoAction(FlowControl fc, Act act)
 
         public bool GotoNextAction()
         {
-            return CurrentBusinessFlow.CurrentActivity.Acts.MoveNext();
+            if (CurrentBusinessFlow.CurrentActivity.Acts.Count - 1 > CurrentBusinessFlow.CurrentActivity.Acts.IndexOf((Act)CurrentBusinessFlow.CurrentActivity.Acts.CurrentItem))
```

Comment:
we are already checking IsLactAction when calling this function. do we really need this condition here?